public/js/screenshot-protection.js

```
@@ -306,39 +386,139 @@
+          triggerDevToolsProtection();
+        }
+      } catch (e) {
+        // Silent fail
+      }
+    }
+
+    // Method 4: Debugger detection
+    function checkDebugger() {
+      try {
+        const start = performance.now();
+        debugger; // This will pause if DevTools is open
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)